src/Sylius/Component/Review/README.md

```diff
@@ -0,0 +1,46 @@
+Review Component [![Build status...](https://secure.travis-ci.org/Sylius/Review.png?branch=master)](http://travis-ci.org/Sylius/Report)
+=================
+
+
+Sylius
+------
+
+Modern ecommerce for PHP and Symfony2. Visit [Sylius.org](http://sylius.org).
+
+Documentation
+-------------
+
+Documentation is available on [**docs.sylius.org**](http://docs.sylius.org/en/latest/components/Report/index.html).
+
+Contributing
+------------
+
+All instructions for contributing to Sylius can be found in the [Contributing Guide](http://docs.sylius.org/en/latest/contributing/index.html).
+
+Support
+-------
+
+If you have a question regarding the usage of this library, please ask on
+[Stackoverflow](http://stackoverflow.com). You should use "sylius"
+tag when posting and make sure to [browse existing questions](http://stackoverflow.com/questions/tagged/sylius).
+
+Sylius on Twitter
+-----------------
+
+[Follow the official Sylius account on Twitter!](http://twitter.com/Sylius).
+
+Bug Tracking
+------------
+
+If you find a bug, please refer to the [Reporting a Bug](http://docs.sylius.org/en/latest/contributing/code/bugs.html) guide.
+
+MIT License
+-----------
+
+License can be found [here](https://github.com/Sylius/Report/blob/master/LICENSE).
+
+Authors
+-------
+
+The component was originally created by [Paweł Jędrzejewski](http://pjedrzejewski.com).
```

Comment:
Despite the fact, that it's ``Component``, not ``Bundle``, everything is ok :smile: